Sources/NetworkProtection/Networking/NetworkProtectionClient.swift

```diff
@@ -236,3 +235,14 @@ extension URL {
     }
 
 }
+
+extension TunnelSettings.SelectedEnvironment {
+    var endpointURL: URL {
+        switch self {
+        case .production:
+            return URL(string: "https://controller.netp.duckduckgo.com")!
+        case .staging:
+            return URL(string: "https://staging1.netp.duckduckgo.com")!
+        }
+    }
+}
```

Comment:
Why not have this directly in `SelectedEnvironment`?

It feels like having this in `NetworkProtectionClient.swift` is only making it less discoverable if we want to use this from elsewhere.